components/VideoPlayer/common/src/commonMain/kotlin/org/jetbrains/compose/videoplayer/VideoPlayer.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.compose.videoplayer
+
+import androidx.compose.runtime.Composable
+
+@Composable
+fun VideoPlayer(url: String, width: Int, height: Int) {
+    VideoPlayerImpl(url, width, height)
+}
+
+expect fun VideoPlayerImpl(url: String, width: Int, height: Int)
```

Comment:
internal?

Reply:
Yep, think alike, already fixed that in update :).